api/src/test/kotlin/org/jetbrains/kotlinx/dl/api/inference/keras/ActivationLayersImportExportTest.kt

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2022 JetBrains s.r.o. and Kotlin Deep Learning project contributors. All Rights Reserved.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlinx.dl.api.inference.keras
+
+import org.jetbrains.kotlinx.dl.api.core.Sequential
+import org.jetbrains.kotlinx.dl.api.core.activation.Activations
+import org.jetbrains.kotlinx.dl.api.core.initializer.HeNormal
+import org.jetbrains.kotlinx.dl.api.core.layer.activation.*
+import org.jetbrains.kotlinx.dl.api.core.layer.core.ActivationLayer
+import org.jetbrains.kotlinx.dl.api.core.layer.core.Input
+import org.jetbrains.kotlinx.dl.api.core.regularizer.L2
+import org.junit.jupiter.api.Test
+
+class ActivationLayersImportExportTest {
+    @Test
+    fun `activation layer`() {
+        LayerImportExportTest.run(
+            Sequential.of(
+                Input(100),
+                ActivationLayer(name = "test_activation_layer", activation = Activations.Exponential)
+            )
+        )
+    }
+
+    @Test
+    fun elu() {
+        LayerImportExportTest.run(
+            Sequential.of(
+                Input(100),
+                ELU(name = "test_elu", alpha = 0.5f)
+            )
+        )
+    }
+
+    @Test
+    fun `leaky relu`() {
```

Comment:
Could you please explain or refer to a rule with test namings, somewhere you use short names without `` and somewhere small sentences in ` `.
I suggest using one type of naming according the previous codebase

Reply:
It is a bit inconsistent, I agree. I looked at the codebase, and it seems that camel-case naming is more prevalent, although there are tests which use spaces (like `org.jetbrains.kotlinx.dl.api.inference.keras.ModelLoaderTest#load initializers from file`). So I changed test function names in this PR so that they use camel-case.